### PR TITLE
Fix test-runs WCT tests

### PR DESCRIPTION
**Pull request by lukebjerring**
## Description
Cleans up some redundant `Promise.resolve(true)` code, and removes the `ready()` test (we don't call ready anymore).

### CODE REVIEW

File: webapp/test/test-runs.html [hunk 1]
```diff
@@ -35,15 +35,15 @@
...
+              resolve();
             }
           }, 0);
         });
       }
 
       setup(() => {
         sandbox = sinon.sandbox.create();
-        sandbox.stub(window, 'fetch', () => Promise.resolve(new Response(fetches[0])));
+        sandbox.stub(window, 'fetch', () => Promise.resolve(new Response(fetches)));
```
Comment (Hexcles) [inline webapp/test/test-runs.html]: Why did you remove `[0]`?

Comment (lukebjerring) [reply webapp/test/test-runs.html]: It's fetching `/api/runs` which returns an array, so the result should be the array. Not sure why it was a single item originally.

**Comment by lukebjerring** [comment]
Will merge after #546 for simplicity.
